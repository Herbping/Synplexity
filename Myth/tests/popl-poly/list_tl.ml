type 'a list =
  | Nil
  | Cons of 'a * 'a list

let list_tl : 'a list -> 'a list |>
 /\(\/([], ['a]) -> [],
    ['a; a] -> [a]) = ?
