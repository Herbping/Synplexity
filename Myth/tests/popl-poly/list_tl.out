(* Generated Thu Oct  8 20:27:03 2015 *)
fun (l1:'a list) ->
  match l1 with
  | Nil u1 -> Nil
  | Cons t1 -> #2 t1
